src/Directory.Build.targets

```
@@ -64,7 +64,7 @@
       <_TestResultsXmlPath>$(_TestOutPathNoExt).xml</_TestResultsXmlPath>
       <_TestResultsHtmlPath>$(_TestOutPathNoExt).html</_TestResultsHtmlPath>
       <_RunOnCore>false</_RunOnCore>
-      <_RunOnCore Condition="$(TargetFramework.StartsWith('netcoreapp'))">true</_RunOnCore>
+      <_RunOnCore Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">true</_RunOnCore>
```

Aren't we reversing the condition here?

---

Ops, yeah sorry. Copied from roslyn PR and forgot to update

```suggestion
      <_RunOnCore Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">true</_RunOnCore>
```

Not sure how this didn't cause any CI failures though.